includes/amp-helper-functions.php

```diff
@@ -151,6 +151,15 @@ function () {
 		do_action( 'amp_plugin_update', $old_version );
 		AMP_Options_Manager::update_option( Option::VERSION, AMP__VERSION );
 	}
+
+	add_action(
+		'rest_api_init',
```

Comment:
I think this be better suited in `AMP_Options_Manager::init()`.

Reply:
That runs on `admin_init`, though, which is after `rest_api_init`.

Reply:
Ah right, seems OK then.